src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorArgumentCollection.java

```diff
@@ -103,7 +103,7 @@
             fullName = FuncotatorArgumentDefinitions.LOOKAHEAD_CACHE_IN_BP_NAME,
             optional = true,
             minValue = 0,
-            doc = "Number of base-pairs to cache when querying variants."
+            doc = "Default number of base-pairs to cache when querying variants.  Can be overridden in individual data source configuration files."
```

Comment:
I don't think it is the default number.  It is the actual number.  It is just initialized to the default number, right?

I think "Number of base-pairs to cache when querying variants.  Can be overridden in individual data source configuration files." is more accurate.

Reply:
Sure.